common/pb/session_commands.proto

```diff
@@ -165,6 +166,8 @@ message Command_AccountPassword {
     }
     optional string old_password = 1;
     optional string new_password = 2;
+    optional string hashed_old_password = 3;
```

Comment:
This shouldn't be here based on our discussion

Reply:
I havent taken the time to work on this because I'd like to discuss alternatives first, note that you can use the hashed password to log in and change the email address in the current implementation